README.md

```diff
@@ -120,6 +111,24 @@ jobs:
 
 **Optional** If set to `${{ github.token }}`, gives HawkScan Action a temporary GitHub API token to enable uploading SARIF data. This input is required if `codeScanningAlerts` is set to `true`.
 
+### `Debugging` 
```

Comment:
Omar added `debug` and `verbose` inputs for this Action, so they should each get their own section rather than using environment variables in this debugging section.